armi/nuclearDataIO/cccc/dlayxs.py

```diff
@@ -224,11 +224,37 @@ def __init__(self, fileName, fileMode, dlayxs):
         self.metadata = dlayxs.metadata
 
     def readWrite(self):
-        """Read and write DLAYXS files.
+        r"""Read and write DLAYXS files.
```

Comment:
Does `r` need to be here?

Reply:
Yes, due to the `\` in `MC\ :sup:`2`-3`.

Reply:
Yup! Agreed.

Reply:
Ah ha!